index.html

```diff
@@ -227,7 +227,20 @@ <h3>
         </ul>
       </div>
     </section>
-
+    <section id="contact">
+      <footer>
+          <h2>Contact me</h2>
+          <p id="para">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          <form action="https://drive.google.com" method="post">
```

Comment:
Kindly change the `action` to be similar to `https://formspree.io/f/{form_id}` you can generate your `{form_id}` using this site provided in the project requirement. [link](https://formspree.io/html).